src/style.css

```diff
@@ -0,0 +1,3 @@
+body {
+  font-family: 'League Spartan', sans-serif;
+}
```

Comment:
Instead of Adding Font to the body, Add it to * (representing everything) because later on, when we add forms or any input field, their font will be the default unless the font is specified in *.